specification/standard_data_types/standard_data_types.tex

```diff
@@ -0,0 +1,20 @@
+\chapter{List of standard data types}
+
+This chapter contains the full list of standard data types defined by the UAVCAN specification.
+The source text of the DSDL data type definitions provided here is also available via the
+official project website at \href{http://uavcan.org}{uavcan.org}.
+
+Regulated non-standard definitions\footnote{I.e., public definitions contributed by vendors and other users
+of the specification, as explained in section \ref{sec:basic_concepts_data_type_regulation}.}
+are intentionally not included in this list, as they are of limited importance for the specification.
```

Comment:
We should find a nicer way to phrase this. While semantically true, it's not polite to tell vendors that their carefully designed datatypes are of limited importance.